app/scripts/auth/controller.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.examples[
+  '$scope',
+  'loginService',
+  function loginControler($scope, loginService) {
+    $scope.user = {
+      email: '',
```

Comment:
Yes, this should be changed in markup. REST API needs 'email' field in order to authenticate user on backend